includes/Classifai/Providers/Azure/ComputerVision.php

```diff
@@ -48,32 +51,60 @@ public function can_register() {
 		if ( empty( $options ) ) {
 			return false;
 		}
+
 		return true;
 	}
 
 	/**
 	 * Register the functionality.
 	 */
 	public function register() {
-		add_filter( 'wp_generate_attachment_metadata', [ $this, 'process_image' ], 10, 2 );
+		add_filter( 'wp_generate_attachment_metadata', [ $this, 'smart_crop_image' ], 10, 2 );
+		add_filter( 'wp_generate_attachment_metadata', [ $this, 'generate_image_alt_tags' ], 10, 2 );
```

Comment:
In my test, changing these filters priority to `8` get image tagging and smart cropping work with Azure Storage which hooks to the same filter with priority set to `9`.